src/include/miopen/mlo_internal.hpp

```diff
@@ -119,6 +119,8 @@ inline int AlignUp(int val, unsigned step)
     return static_cast<int>(((static_cast<unsigned>(val) + step - 1) / step) * step);
 }
 
+inline size_t AlignUp(size_t num, size_t align) { return (num + align - 1) / align * align; }
```

Comment:
Are we ok allowing the possible div0 if align==0? The function does not check for this either. They should be consistent, so I'll let you decide whether to ignore it or modify both.

Reply:
fixed it.